src/locales/es/battler-tags.json

```
@@ -1 +1,71 @@
-{}
\ No newline at end of file
+{
+  "trappedDesc": "trampa",
+  "flinchedDesc": "retroceso",
+  "confusedDesc": "confusión",
+  "infatuatedDesc": "enamoramiento",
+  "seedDesc": "drenado",
+  "nightmareDesc": "pesadillas",
+  "ingrainDesc": "raíces",
+  "drowsyDesc": "sueño",
+  "rechargingLapse": "¡{{pokemonNameWithAffix}} necesita\nrecuperarse de su ataque!",
+  "trappedOnAdd": "¡{{pokemonNameWithAffix}} no puede escapar!",
+  "trappedOnRemove": "¡{{pokemonNameWithAffix}} se ha\nliberado de {{moveName}}!",
+  "flinchedLapse": "¡{{pokemonNameWithAffix}} se amedrentó!",
+  "confusedOnAdd": "¡{{pokemonNameWithAffix}} se encuentra confuso!",
+  "confusedOnRemove": "¡{{pokemonNameWithAffix}} ya no está confuso!",
+  "confusedOnOverlap": "¡{{pokemonNameWithAffix}} ya está confuso!",
+  "confusedLapse": "¡{{pokemonNameWithAffix}} está confuso!",
+  "confusedLapseHurtItself": "¡Está tan confuso que se ha herido a sí mismo!",
+  "destinyBondLapseIsBoss": "Mismo Destino no afecta a {{pokemonNameWithAffix}}.",
+  "destinyBondLapse": "¡{{pokemonNameWithAffix2}} ha sufrido\nel mismo destino que {{pokemonNameWithAffix}}!",
+  "infatuatedOnAdd": "¡{{pokemonNameWithAffix}} se ha enamorado\nde {{sourcePokemonName}}!",
+  "infatuatedOnOverlap": "¡{{pokemonNameWithAffix}} ya está enamorado!",
+  "infatuatedLapse": "¡{{pokemonNameWithAffix}} se ha enamorado\ndebido a {{sourcePokemonName}}!",
+  "infatuatedLapseImmobilize": "¡El enamoramiento impide\nque\n{{pokemonNameWithAffix}}reaccione!",
```

```suggestion
  "infatuatedLapseImmobilize": "¡El enamoramiento impide que\n{{pokemonNameWithAffix}} reaccione!",
```
Removed a line break